portfolio/src/main/webapp/discussionScript.js

```diff
@@ -1,7 +1,29 @@
 
 function displayList() {
   fetch('/discussion').then((response) => response.json()).then((data) => {
-    document.getElementById('discussion-container').textContent =
-        data.join(', ');
+
+
+    let posts = document.createElement('div');
+
+    for (let i = 0; i < data.length; ++i) {
+      let title = document.createElement('div');
+      title.id = "post-title";
```

Comment:
You're going to end up with a bunch of elements with the same id by doing this.  You should either use a class instead if you want to be able to refer to all of them, or else use somthing like `"post-title-" + i` to ensure that the id is unique

Reply:
ah. good catch didn't think about that